src/ui/public/visualize/visualize_legend.js

```diff
@@ -48,13 +50,16 @@ uiModules.get('kibana')
         handler.unHighlight.call(el, handler.el);
       };
 
-      $scope.setColor = function (label, color) {
+      $scope.setColor = function (label, color, event) {
         const colors = $scope.uiState.get('vis.colors') || {};
         if (colors[label] === color) delete colors[label];
         else colors[label] = color;
         $scope.uiState.setSilent('vis.colors', null);
         $scope.uiState.set('vis.colors', colors);
         $scope.uiState.emit('colorChanged');
+        if (event) {
+          event.currentTarget.blur();
```

Comment:
Out of curiosity, why are we doing this?

Reply:
Good question. I was tinged with the slight idea, of not having the color button keeping the focus ring and it's then selected state. But honestly, we don't do this anywhere else, and again I want to replace this by the unified color picker in the future, so yeah I will throw it out for this PR.